variable/rebin.cpp

```diff
@@ -30,6 +30,8 @@ void rebin_non_inner(const Dim dim, const VariableConstView &oldT,
   // coord is 1D.
   int iold = 0;
   int inew = 0;
+  const bool is_bool = newT.dtype() == dtype<bool>;
+  // const bool is_bool = newT.dtype().index == std::type_index(typeid(bool));
```

Comment:
commented code.

Reply:
yes this is still part of my debugging for the windows build...

Reply:
Does it work if you put the namespace, i.e., `core::dtype<bool>`?

Reply:
No.
I’ve done this workaround, let me know if it is a bad idea https://github.com/scipp/scipp/pull/1198/commits/53894509fba8abeb71a4c718fc652094caa3ab43